content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,158 @@
+
+# Prerequisites
+
+- An installed Private Cloud Gateway (PCG). Review [Install and Manage MAAS Gateway](/clusters/data-center/maas/install-manage-maas-pcg) for guidance.
+
+
+- At least one preconfigured MAAS cloud account with credentials.
+
+
+- A cluster profile for the MAAS environment. Review [Cluster Profiles](/cluster-profiles) for more information. 
+
+# Deploy a MAAS cluster
+
+To deploy a new MAAS cluster:
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. Then click the **Add New Cluster** button.
+
+
+3. Click **Deploy New Cluster** on the Create a New Cluster page.
+
+
+4. Select **MAAS** and click the **Start MAAS Configuration** button.
+
+
+5. Provide basic cluster information: **Cluster name**, **Description**, and **Tags**.
+
+
+6. Select your cloud account from the **drop-down Menu** and click **Next**.
+
+
+7. Select the cluster profile for your MAAS cluster. 
+
+
+8. Review and override pack parameters as desired and click **Next**. By default, parameters for all packs are set with values defined in the cluster profile.
+
+
+9. Select a MAAS cloud account and provide placement information.
+
+
+10. Select a domain from the **Domain drop-down Menu** and click **Next**. 
+
+
+11. Configure the master and worker node pools. Enter the values for properties listed in the following tables.
+
+<br />
+
+
+#### Master Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| Name | A descriptive name for the node pool. |
+| Size | Number of VMs to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5. |
+| Allow worker capability | Select this option to allow workloads to be provisioned on master nodes. |
+| [Labels](/clusters/cluster-management/taints#overviewonlabels) | Labels apply placement constraints on a pod. For example, you can add a label to make a node eligible to receive the workload. |
+| [Taints](/clusters/cluster-management/taints#overviewontaints) | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. |
+| Instance type | The compute instance type to be used for all nodes in the node pool. |
+| Availability Zones | These are multiple, isolated locations within a Region. When you choose one or more availability zones, Palette provides fault tolerance to guard against hardware failures and network failures by provisioning nodes across availability zones if multiple zones are selected. |
+| Disk Size | Specify the required storage size. |
+
+<br />
+
+
+#### Worker Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| Name | A descriptive name for the node pool. |
+| Size | Number of VMs to be provisioned for the node pool.|
+| Rolling Update | Rolling update has two available options. Review the Update Parameter list below for more details.|
```

Comment:
@ritawatson let's break this update list out to its own table like we have done for other sub-settings before.

Reply:
@karl-cardenas-coding tracked down the info and created a table for these.